stactools_landsat/stactools/landsat/commands.py

```diff
@@ -1,16 +1,45 @@
+import json
+import os
+import sys
+
 import click
+from pystac import Item
+from stactools.landsat.utils import (transform_mtl_to_stac,
+                                     transform_stac_to_stac)
+
 
+def create_landsat_command(cli):
+    @cli.command("landsat-convert",
+                 short_help="Convert a Landsat MTL file to a STAC Item")
+    @click.option("--mtl", "-m", help="Path to an MTL file.")
+    @click.option("--stac", "-s", help="Path to a STAC file.")
+    @click.option(
+        "--enable-proj",
+        "-p",
+        default=True,
+        is_flag=True,
+        help="Enable the proj extension. Requires access to blue band.")
+    @click.option("--dst", "-d", help="Output directory")
+    def landsat_command(mtl, stac, enable_proj, dst):
+        if mtl and stac or (not mtl and not stac):
+            print("Please choose one of either MTL or STAC, not both")
+            sys.exit(1)
 
-def convert_mtl_to_item(mtl, dst):
-    print('TODO: LANDSAT CONVERT {} {}'.format(mtl, dst))
+        item = None
+        if mtl:
+            # Transform not implemented, so tell folks
+            print("MTL transform not yet implemented.")
+            sys.exit(1)
+            with open(mtl) as f:
+                item = transform_mtl_to_stac(json.load(f))
+        elif stac:
+            with open(stac) as f:
+                in_item = Item.from_dict(json.load(f))
```

Comment:
Prefer `pystac.read_file`. You can check that the read-in stac object is an Item by checking

`in_item.STAC_OBJECT_TYPE == pystac.STACObjectType.ITEM`

Reply:
I changed to using pystac.Item.from_file, which works fine. Using pystac.from_file does _not_ work, but I don't want to dive into why currently!